src/components/MainNav/CurrentApp/CurrentApp.js

```diff
@@ -31,13 +32,25 @@ const defaultProps = {
 const CurrentApp = ({ currentApp, iconData, id, badge }) => {
   const { displayName, description, module } = currentApp;
   const iconKey = module && module.replace(/^@[a-z0-9_]+\//, '');
+  let Element = 'div';
+  const elementProps = {};
```

Comment:
I hate that this is a `const` that we mutate. Probably just eslint being overzealous about `const` use?

Reply:
I updated to this to reduce the nails-on-the-`const`board effect:
```
let elementProps = {};

if (currentApp && (currentApp.home || currentApp.route)) {
  Element = Link;
  elementProps = { to: (currentApp.home || currentApp.route) };
}
```

Reply:
And yes, it was ESLint being silly.